### ISSUE TITLE
Error: corrupted size vs. prev_size when using python3.6

### ISSUE THREAD

**Issue by NewEricWang** [tensorflow/tensorflow#21645]
Hi, 
I install tensorflow-1.9; then install pytorch=0.4 at python2 and python3 respectively.
I use python2 to run my code, no error is reported; but when using python3 to run it, an error is report after finishing run. The error like as follows:
Error in `/restools/tools/miniconda/miniconda3/bin/python': corrupted size vs. prev_size: 0x000055b2bcb5c690 ***

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by NewEricWang** [comment]
Have I written custom code:  No, I use r9y9's wavenet_vocoder.
OS Platform and Distribution: Ubuntu 14.0
TensorFlow installed from : anaconda3/pip
TensorFlow version: 1.9 + pytorch=0.4
Bazel version: I don't know it.
CUDA/cuDNN version: cuda8.0-cudnn6.0
GPU model and memory: tina 1080i, mem=12G
Exact command to reproduce ： /restools/tools/miniconda/miniconda3/bin/python synthesis_test.py --preset=./presets/v1.0.json --conditional=./180k/mel-wnet/speech-mel-00005.npy --file-name-suffix=00005  ./works/v1/checkpoint_step000740000_ema.pth ./works/0805_v1_180k_740kv1
Mobile device : no

**Comment by tensorflowbutler** [comment]
Nagging Assignee @karmel: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @karmel: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @karmel: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by karmel** [comment]
Can you retry with the most recent release of TF?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by harshini-gadige** [comment]
@NewEricWang  - Does this still persist with the latest version of TF ?

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by towhid355** [comment]
Code stop after showing this message. 
```
tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
corrupted size vs. prev_size
```
Python 3.8
tensorflow==2.2.0

**Comment by pestrstr** [comment]
For anyone still struggling with this problem, I have found out that, if you are using this old version of TF (TFv1) on gpus together with PyTorch, importing and setting TFv1 on gpus before importing PyTorch or PyTorch-based modules solved my issue. 
Still, I cannot tell which is the exact problem.
